### DISCUSSION TITLE
Prebuild seems to hang for 5-10 minutes

### DISCUSSION THREAD

**Discussion by billy1kaplan** [community/community#106678] [Codespaces]
### Select Topic Area

Bug

### Body

I'm seeing prebuilds that seem to hang without output for 5-10 minutes. Running `devcontainer build --workspace-folder .` locally does not have the same issue.

<img width="896" alt="image" src="https://github.com/community/community/assets/12505097/1dbe06a5-7417-4cc1-a038-fcb3c79519b9">

**Comment by CharlieDigital** [comment]
Related: https://github.com/orgs/community/discussions/106678
